src/js/components/mapWidgets/widgetContent/coordinatesDDSection.tsx

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { RootState } from 'js/store/index';
 
 import { DDSectionProps } from 'js/interfaces/coordinateForm';
 
+import { coordinatesContent } from 'configs/modal.config';
+
 import { ReactComponent as TrashCanIcon } from 'images/trashCanIcon.svg';
 
 export default function DDSection(props: DDSectionProps): JSX.Element {
+  const selectedLanguage = useSelector(
```

Comment:
we can get selected Language info from parent component and just pass down language as prop, reducing number of `useSelectors` is a good goal have if that is reasonable.